packages/gatsby-source-wordpress/package.json

```diff
@@ -8,8 +8,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.20.13",
-    "@rematch/core": "^1.4.0",
-    "@rematch/immer": "^1.2.0",
+    "@rematch/core": "^2.2.0",
+    "@rematch/immer": "^2.1.3",
```

Comment:
The old version of rematch had completely broken TS types

Reply:
nice! it did indeed. I looked at upgrading around a year ago and went through the whole process to discover rematch was no longer working and I didn't have more time to figure out why 🤦 hopefully you fared better here!

Reply:
good thing is, while they're odd, the tests are fairly comprehensive, so if there's any problem it'll definitely show up in the tests